CX/Telegram/server.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const projectId = 'my-project';
+// const location = 'global';
+// const agentId = 'my-agent';
+// const languageCode = 'en'
+// const TELEGRAM_TOKEN='1234567898:ABCdfghTtaD8dfghdfgh45sdf65467M';
+// const SERVER_URL='https://example.com';
+
+const express = require('express');
+const axios = require('axios');
+const bodyParser = require('body-parser');
+
+const API_URL = `https://api.telegram.org/bot${TELEGRAM_TOKEN}`;
+const URI = `/webhook/${TELEGRAM_TOKEN}`;
+const WEBHOOK = SERVER_URL + URI;
+
+const app = express();
+app.use(bodyParser.json());
+
+
+// Imports the Google Cloud Some API library
+const {SessionsClient} = require('@google-cloud/dialogflow-cx');
+/**
+ * Example for regional endpoint:
+ *   const location = 'us-central1'
+ *   const client = new SessionsClient({apiEndpoint: 'us-central1-dialogflow.googleapis.com'})
+ */
+const client = new SessionsClient();
```

Comment:
It might be easily missed that this also needs changes to work. Is it possible to generate the regional Dialogflow endpoint from `locationId` variable?